setup.py

```diff
@@ -14,10 +40,15 @@ def readme():
         'Development Status :: 4 - Beta',
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
         'Topic :: Scientific/Engineering :: Astronomy',
       ],
       keywords='astronomy astrophysics image display',
       url='http://js9.si.edu',
       license='MIT',
-      packages=['pyjs9'],
-      zip_safe=False)
+      packages=['pyjs9', 'pyjs9.extern'],
+      zip_safe=False,
+      tests_require=['pytest', 'selenium'],
```

Comment:
I think that the best would be to add 

    install_requires = ['requests', 'six']

and get rid of ``pyjs9/extern`` altogether